pkg/envctx/env_context.go

```diff
@@ -46,7 +46,7 @@ type ChartDetails struct {
 	Repository string
 }
 
-// ResolveChartRepository resolves the chart details from a full or local name and an optional repository URL.
+// ChartDetails resolves the chart details from a full or local name and an optional repository URL.
```

Comment:
Same as above squash this commit into [`feat: add support for helmfile in the apps model`](https://github.com/jenkins-x/jx/pull/6660/commits/ce6ab1c44b5eea7208843c5f6064e478f1b707d7#diff-db1ba19080e5965ef5eb75547a23b90dR43)